--- linux-master/drivers/video/jz4770_fb.c	2019-12-16 18:09:47.616690939 +0100
+++ linux-master/drivers/video/jz4770_fb.c	2019-12-16 18:03:04.819392937 +0100
@@ -84,7 +84,8 @@
 		LCD_CFG_HSP |	/* Hsync polarity: active low */
 		LCD_CFG_VSP,	/* Vsync polarity: leading edge is falling edge */
 	/* bw, bh, dw, dh, fclk, hsw, vsw, elw, blw, efw, bfw */
-    320, 240, 320, 240, 60, 50, 1, 10, 70, 5, 5
+  /*320, 240, 320, 240, 60, 50, 1, 10, 70, 5, 5,*/
+	320, 240, 320, 240, 60, 30, 8, 60, 20, 8, 2,
 	/* Note: 432000000 / 72 = 60 * 400 * 250, so we get exactly 60 Hz. */
 };
 
@@ -1424,4 +1425,4 @@
 
 MODULE_DESCRIPTION("Jz4770 LCD frame buffer driver");
 MODULE_AUTHOR("Maarten ter Huurne <maarten@treewalker.org>");
-MODULE_LICENSE("GPL");
\ No hay ningún carácter de nueva línea al final del fichero
+MODULE_LICENSE("GPL");
